app/views/components/_related-navigation-sidebar.html.erb

```diff
@@ -0,0 +1,45 @@
+<%
+  defined_sections = ["topics", "publishers", "collections", "policies"]
+  related_content = [
+    {"related_items" => related_items ||= [] },
+    {"topics" => topics ||= [] },
+    {"publishers" => publishers ||= [] },
+    {"collections" => collections ||= [] },
+    {"policies" => policies ||= [] },
+  ]
+  other ||= []
+  format_data_for_related_navigation(related_content, other)
+%>
+
+<% if related_content.map(&:values).flatten.any? || other.any? %>
+  <aside class="app-c-related-navigation-sidebar" role="complementary">
+    <h2 class="app-c-related-navigation-sidebar__main-heading" >
+      <%= t('components.navigation_sidebar.related_content') %>
+    </h2>
+
+    <% related_content.each do |section| %>
+      <% section.each do |section_title, links| %>
+        <% if links.any?%>
+          <% unless section_title === "related_items" %>
+            <h3 class="
+              app-c-related-navigation-sidebar__sub-heading
```

Comment:
You could build this string of classes up in the erb block above and call a variable here.